infrastructure/modules/psql/variables.tf

```
@@ -0,0 +1,315 @@
+variable "psql_SubnetCidr" {
+  type        = string
+  description = "The CIDR range to use for psql subnet"
+}
+
+variable "organization" {
+  type        = string
+  description = "The organization/service owner name"
+}
+
+variable "environment" {
+  type        = string
+  description = "The environment (ATxx/TTx/YTx)"
+}
+
+variable "location" {
+  type        = string
+  default     = "norwayeast"
+  description = "The location where the resources will be placed."
+}
+variable "psql_NetworkName" {
+  type        = string
+  description = "psql network name"
+}
+
+variable "psql_NetworkResourceGroup" {
+  type        = string
+  description = "psql network name"
+}
+
+variable "psql_ResourceGroup" {
+  type        = string
+  description = "psql resource group"
+}
+
+variable "product" {
+  type        = string
+  description = "Product the resource belongs to"
+}
+
+variable "psql_ComputeSize" {
+  type        = string
+  description = "psql compute size"
+}
+
+variable "psql_HighAvailability" {
+  type        = bool
+  description = "Enable psql high availability"
+}
+
+variable "locks_off" {
+  type        = bool
+  description = "If locks should be on or off"
+}
+
+variable "psql_AdminGroups" {
```

Keep to one type of formating, for terraform i prefer snake_case
```suggestion
variable "psql_admin_groups" {
```